rskj-core/src/main/java/co/rsk/mine/MinerUtils.java

```diff
@@ -202,6 +203,9 @@
                 accountNonces.put(txSender, txNonce);
 
                 logger.debug("Accepted tx={} sender: {} value: {} nonce: {}", hash, txSender, txValue, txNonce);
+            } catch (NodeReferenceException nre) {
```

Comment:
Cannot spot this.. What method in this try-catch block can throw `NodeReferenceException`?

Reply:
see it now) probably `originalRepo.getNonce`

Reply:
Yep, that's the one. It is very entangled and difficult to see what's happening, but it was the only way I saw to stop the node back at the `Start.java` class.

Reply:
Outdated